Document new option `start_position` for logs (#6941)
DataDog/documentation: content/en/agent/logs/_index.md

| `include_units`  | No       | If `type` is **journald**, list of the specific journald units to include.                                                                                                                                                                                                                                                                              |
| `exclude_units`  | No       | If `type` is **journald**, list of the specific journald units to exclude.                                                                                                                                                                                                                                                                              |
| `sourcecategory` | No       | A multiple value attribute used to refine the source attribute, for example: `source:mongodb, sourcecategory:db_slow_logs`.                                                                                                                                                                                                                             |

> | `start_position` | No       | If `type` is **file**, set the position for the Agent to start reading the file. Valid values are `beginning` and `end`. If `path` contains a wildcard character, `beginning` is not supported.                                                                                                                                                   |

Reviewer: let's mention the default value and in which agent version this was added:

```suggestion
| `start_position` | No       | If `type` is **file**, set the position for the Agent to start reading the file. Valid values are `beginning` and `end` (default: `end`). If `path` contains a wildcard character, `beginning` is not supported. _Added in Agent v6.19/v7.19_                                                                                                                                                  |
```

> | `start_position` | No       | If `type` is **file**, set the position from where the agent will start reading the file. Valid values are `beginning` and `end`. If `path` contains wildcard character `beginning` is not supported.                                                                                                                                                   |

Reviewer 2: ```suggestion
| `start_position` | No       | If `type` is **file**, set the position for the Agent to start reading the file. Valid values are `beginning` and `end`. If `path` contains a wildcard character, `beginning` is not supported.                                                                                                                                                   |
```
